### DISCUSSION TITLE
Ch #1 Exercise on file://

### DISCUSSION THREAD

**Discussion by smriti-spotnana** [browserengineering/book#955] [Q&A] [answered]
How can my browser show a local file's content ? - won't socket look for the IP address of the server serving the url (file:// etc in this case) first .. and I have not setup my machine to serve this file in anyway .. so how would the socket locate this file I am thinking

**Comment by chrishtr** [comment, marked as answer]
Hint: You'll need to parse the URL and use python file APIs to load the file directly.
